### PR TITLE
fix: theming css props use stylesheet instead of inline on html el

### PR DESCRIPTION
**Pull request by jorenbroekema**

### CODE REVIEW

File: themes/src/theme.js [hunk 1]
```diff
@@ -66,8 +66,37 @@ export const setTheme = (_theme, _colors) => {
...
   };
 
+  let simbaPropsSheetInner = ':root {';
   for (let [key, value] of Object.entries(obj)) {
-    document.documentElement.style.setProperty(key, value);
+    simbaPropsSheetInner += `${key}:${value};\n`;
+  }
+  simbaPropsSheetInner += '}';
+
+  // Constructible stylesheets not supported on Safari
```
Comment (gqio) [inline themes/src/theme.js:75]: there is no polyfill for that ?

**Review by gqio — APPROVED** [review]

File: themes/src/theme.js (hunk 1 above)
Comment (jorenbroekema) [reply themes/src/theme.js:75]: https://www.npmjs.com/package/construct-style-sheets-polyfill there is, but it's too large for something I can workaround with <10 LOC and no degradation perf-wise.

Comment (gqio) [reply themes/src/theme.js:75]: I was thinking about a polyfill made by Webreflection but ok.

**Comment by jorenbroekema** [comment]
Oh, you have a link to it? Didn't know he also made one

**Pull request merged**